homeassistant/components/azure_data_explorer/strings.json

```
@@ -3,15 +3,18 @@
     "step": {
       "user": {
         "title": "Setup your Azure Data Explorer integration",
-        "description": "Enter connection details.",
+        "description": "Enter connection details.\n\n'Use queued ingestion' must be checked when using ADX free cluster.",
         "data": {
-          "cluster_ingest_uri": "Cluster ingest URI",
-          "database": "Database name",
-          "table": "Table name",
+          "cluster_ingest_uri": "Cluster Ingest URI",
+          "authority_id": "Authority ID",
           "client_id": "Client ID",
           "client_secret": "Client secret",
-          "authority_id": "Authority ID",
+          "database": "Database name",
+          "table": "Table name",
           "use_queued_ingestion": "Use queued ingestion"
+        },
+        "data_description": {
+          "cluster_ingest_uri": "Ingest-URI of the cluster"
```

```suggestion
          "cluster_ingest_uri": "Ingest-URI of the cluster",
          "use_queued_ingestion": "Must be checked when using ADX free cluster."
```

Can we detect automatically if it's a free cluster or not?

---

Did not know that it is currently not working for checkboxes